peachjam_api/views.py

```diff
@@ -53,3 +57,73 @@ def post(self, request, ingestor_id):
 
             return Response({"data": serializer.data, "ingestor_id": ingestor_id})
         return Response(serializer.errors, status=400)
+
+
+class DuplicateForm(forms.Form):
+    court = forms.CharField(max_length=100)
+    date = forms.DateField()
+
+
+class CaseNumberForm(forms.ModelForm):
+    class Meta:
+        model = CaseNumber
+        fields = [
+            "string_override",
+            "string",
+            "number",
+            "year",
+            "matter_type",
+        ]
+        exclude = ["document", "id"]
+
+
+class CheckDuplicatesView(APIView):
+    permission_classes = []
+
+    def post(self, request):
```

Comment:
@longhotsummer  I have kept this as an APIView. FormView is focused on processing a single form and there is no real benefit of using it since we have to subclass the post method. We could use the base django view but we already have APIView and Response classes imported in the module